ui/chatArea/innerStyle.css

```diff
@@ -1,42 +1,9 @@
-p {
-  white-space: pre-wrap;
-}
-
-p.msg {
-  color: @black;
-  font: @big;
-}
-
-p.action {
-  color: #1818FF;
-  font: @big;
-  font-style: italic;
-}
-
-p.typing {
-  color: @mediumGreyLight;
-  font: @big;
-}
-
-span.quote {
-  color: #279419;
-}
-
-p.alert {
-  margin-left: 0px;
-  margin-right: 0px;
-  color: @black;
-  background-color: @orange;
-  font: @big;
-}
-
-p.alert_name {
-  color: @black;
-  background-color: @orange;
-  font: @bigBold;
-}
-
-a {
-    color: blue;
-}
+body {font: @baseFont;}
+p {white-space: pre-wrap;}
+.action {color: #1818FF; font-style: italic;}
+.typing {color: @mediumGreyLight;}
+.quote {color: #279419;}
+.alert {margin-left: 0px; margin-right: 0px; background-color: @orange;}
+.alert_name {background-color: @orange; font: @bigBold;}
+a {color: blue;}
```

Comment:
>>It increases the hurdle to change the style for "inexperienced" people, which comes to another advantage. :)

>Yep, that's pretty much what your change does.

Sorry, I didn't make it really clear: making it harder for people to contribute for no good reason is **wrong**. Please don't do that shit.